src/screen.cc

```diff
@@ -711,12 +733,18 @@ std::string CScreen::choose_string(std::vector<std::string> choices)
 
             if (count == selected)
                 wattron(childwin, A_UNDERLINE | A_STANDOUT);
-            else
-                wattrset(childwin, A_NORMAL);
 
             mvwaddstr(childwin, y, x,  choice.c_str());
+
+            wattrset(childwin, A_NORMAL);
             count += 1;
         }
+        /*
+         * Display the search string in the last line.
+         */
+        if (search_string != "") {
+            mvwprintw(childwin, height - 1, 1, "%s %s", "search:", search_string.c_str());
```

Comment:
I'd change "search:" to "`filter:`" here, and add a trailing space.

      mvwprintw(childwin, height - 1, 1, "%s %s ", "filter:", search_string.c_str());